table/pagination.go

```diff
@@ -31,14 +31,14 @@ func (m *Model) TotalRows() int {
 
 // VisibleIndices returns the current visible rows by their 0 based index.
 // Useful for custom pagination footers.
-func (m *Model) VisibleIndices() (start, end int) {
```

Comment:
I want to be careful about modifying an exported function like this.  `count` should always just be `end-start`, so we should probably just use that internally.

Reply:
Makes sense, I reverted the change.